backend/src/controllers/authController.ts

```diff
@@ -0,0 +1,34 @@
+import clientModel from "../models/client";
+import bcrypt from "bcrypt";
+import { v4 as uuidv4 } from "uuid";
+import { signAccessToken, signRefreshToken } from "../utils/signTokens";
+
+const register = async (req, res) => {
+  try {
+    let client = await clientModel.findOne({ email: req.body.email });
```

Comment:
You can set this as const and save the clientModel immediately (see next comment)
```suggestion
    const client = await clientModel.findOne({ email: req.body.email });
```